src/test/java/array/Array1dTest.java

```diff
@@ -77,7 +77,7 @@ void shouldThrowExceptionCellIsAlreadyOccupied() {
                         Coordinates1d.of(i),
                         10 * i));
         // expect
-        assertThrows(IllegalArgumentException.class,
+        assertThrows(ArrayStoreException.class,
```

Comment:
I think it's pretty common to use IAE if argument is not valid to perform an operation. Why do you want use another exception type?

Reply:
Cause it about storing in an array. And the operation is about storing in array

Reply:
ok